After upgrade to 3.8.7 and master, many projects are failing to load now, due to a JSON

There are no lizmap log messages and if I search these project for cababilities them works fine.
il LWC the pages crashes on map loading/constructing

![Image](https://github.com/user-attachments/assets/3f8a5534-247e-417c-a469-8212d21a4471)

If i use the same layers and try to reconfigure a new project the new project works well!
all these (damaged) projects worked well before the lizmap-server-plugin upgrade each in LWC 3.8.6 and 3.7.14, after the upgrade didnt work anymore on each versions.

```
Versione	3.8.7 - [bee4379f1](https://github.com/3liz/lizmap-web-client/commit/bee4379f1)
Data di rilascio	2025-04-01
URL per il plugin di QGIS desktop	http://mydomain/webgism/
Moduli
Modulo	Versione
pgrouting	1.1.2
QGIS Server
Versione
Numero	[3.40.5 ](https://github.com/qgis/QGIS/releases/tag/final-3_40_5)- [8d6d1b54486](https://github.com/qgis/QGIS/commit/8d6d1b54486)
Nome	Bratislava
QJazz o Py-QGIS-Server	Non installato
Plugins
Plugin	Versione
Data Plotly [Aiuto](https://github.com/ghtmtt/DataPlotly/blob/master/README.md)	[4.2.0 - 5edce82](https://github.com/ghtmtt/DataPlotly/commit/5edce820e580afb870fdf83d9a3f4fa9031a386c)
atlasprint [Aiuto](https://github.com/3liz/qgis-atlasprint)	[3.4.2](https://github.com/3liz/qgis-atlasprint/releases/tag/3.4.2)
Lizmap server [Aiuto](https://docs.lizmap.com/current/fr/install/pre_requirements.html#lizmap-qgis-server-plugin)	[2.13.0](https://github.com/3liz/qgis-lizmap-server-plugin/releases/tag/2.13.0)
wfsOutputExtension [Aiuto](https://github.com/3liz/qgis-wfsOutputExtension)	[1.8.3](https://github.com/3liz/qgis-wfsOutputExtension/releases/tag/1.8.3)

the only error I receive is:

map.js:4029 TypeError: e.Capability.Layer.BoundingBox is not iterable
    at Object.configsloaded (Lizmap.js:78:73)
    at initialize.triggerEvent (OpenLayers.js?_r=250408164551:497:145)
    at map.js:3723:33
    at map.js:3722:42
```
from browser dev tools on 3.8.7

while in 3.7.14 i receive

```
map.js:4029 Error: WMS GetCapabilities not loaded: 400 Bad Request
    at map.js:3677:31
(anonymous)	@	map.js:4029
and this after refresh:

map.js:4127 TypeError: e.Capability.Layer.BoundingBox is not iterable
    at Object.configsloaded (Geolocation.js:24:40)
    at initialize.triggerEvent (OpenLayers.js?_r=250410134938:246:145)
    at map.js:3784:44
```
in 3.7.14 also:

```
Versione	3.7.14 - [87fce24f5](https://github.com/3liz/lizmap-web-client/commit/87fce24f5)
Data	2025-01-07
URL per il plugin di QGIS desktop	http://myhost/wgis/
QGIS Server
Versione
Numero	[3.40.5](https://github.com/qgis/QGIS/releases/tag/final-3_40_5)
Nome	Bratislava
ID del commit	[8d6d1b54486](https://github.com/qgis/QGIS/commit/8d6d1b54486)
Py-QGIS-Server	not used
Plugins
Plugin	Versione
Data Plotly	[4.2.0 - 5edce82](https://github.com/ghtmtt/DataPlotly/commit/5edce820e580afb870fdf83d9a3f4fa9031a386c)
atlasprint	[3.4.2](https://github.com/3liz/qgis-atlasprint/releases/tag/3.4.2)
Lizmap server	[2.13.0](https://github.com/3liz/qgis-lizmap-server-plugin/releases/tag/2.13.0)
wfsOutputExtension	[1.8.3](https://github.com/3liz/qgis-wfsOutputExtension/releases/tag/1.8.3)
```

### Steps to reproduce the issue

``

### Versions, safeguards, check summary etc

**Versions :**

* Lizmap Web Client : 3.8.7 - commit bee4379f1 https://github.com/3liz/lizmap-web-client/commit/bee4379f1
* Lizmap plugin : 4.4.8
* QGIS Desktop : 3.40.5
* QGIS Server : 3.40.5
* Py-QGIS-Server or QJazz : not used
* QGIS Server plugin DataPlotly : 4.2.0
* QGIS Server plugin atlasprint : 3.4.2
* QGIS Server plugin lizmap_server : 2.13.0
* QGIS Server plugin lizmap_server_master : 2.9.5-alpha
* QGIS Server plugin wfsOutputExtension : 1.8.3

<details>
<summary>List of Lizmap Web Client modules :</summary>
<br/>
* pgrouting : 1.1.2
</details>

<details>
<summary>List of safeguards :</summary>
<br/>
* Mode : normal<br/>
* Allow parent folder : yes<br/>
* Number of parent : 10 folder(s)<br/>
* Prevent other drive : no<br/>
* Prevent PG service : no<br/>
* Prevent PG Auth DB : no<br/>
* Force PG user&pass : no<br/>
* Prevent ECW : no<br/>
</details>


### Check Lizmap plugin

- [x] I have done the step just before in the Lizmap QGIS desktop plugin before opening this ticket. Otherwise, my ticket is not considered valid and might get closed.

### Operating system

Ubuntu 22.04

### Browsers

Chrome

### Browsers version

Firefox 137.0.1 (64 bit), Chrome 135.0.7049.84 (Build ufficiale) (a 64 bit) 

### Relevant log output

```shell

```

--- a/assets/src/modules/Lizmap.js
+++ b/assets/src/modules/Lizmap.js
@@ -77,7 +77,7 @@ export default class Lizmap {
                 proj4.defs("CRS:84","+proj=longlat +datum=WGS84 +no_defs +type=crs");
                 register(proj4);
                 // Update project projection if its axis orientation is not ENU
-                if (configProj.ref !== "") {
+                if (configProj.ref !== "" && Array.isArray(wmsCapabilities.Capability.Layer.BoundingBox)) {
                     // loop through bounding boxes of the project provided by WMS capabilities
                     for (const bbox of wmsCapabilities.Capability.Layer.BoundingBox) {
                         // If the BBOX CRS is not the same of the project projection, continue.

